site/specs-temp/numbers.json

```diff
@@ -13430,6 +13430,169 @@
         }
       }
     },
+    "/accounts/{accountId}/shortcodes": {
+      "get": {
+        "tags": [
+          "Short codes"
+        ],
+        "description": "Retrieve shortcodes",
+        "summary": "Retrieve shortcodes",
+        "operationId": "listShortcodes",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Retrieve list shortcodes",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/shortCodeResponseList"
+                }
+              }
+            }
+          },
+          "400": {
+            "$ref": "#/components/responses/badRequestError"
+          },
+          "401": {
+            "$ref": "#/components/responses/unauthorizedError"
+          },
+          "403": {
+            "$ref": "#/components/responses/forbiddenError"
+          },
+          "404": {
+            "$ref": "#/components/responses/notFoundError"
+          },
+          "405": {
+            "$ref": "#/components/responses/notAllowedError"
+          },
+          "409": {
+            "$ref": "#/components/responses/conflictError"
+          },
+          "429": {
+            "$ref": "#/components/responses/tooManyRequestsError"
+          },
+          "500": {
+            "$ref": "#/components/responses/internalServerError"
+          }
+        }
+      }
+    },
+    "/accounts/{accountId}/shortcodes/{shortcode}/{country}": {
+      "get": {
+        "tags": [
+          "Short codes"
+        ],
+        "description": "Retrieve shortcode",
+        "summary": "Retrieve shortcode",
+        "operationId": "getShortcode",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Retrieve shortcode",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/shortCodeResponse"
+                }
+              }
+            }
+          },
+          "202": {
+            "description": "Retrieve shortcode",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/shortCodeResponse"
```

Comment:
I think 202 will return a different body like prob a message that says "Order already in progress" or something like that. We will groom a story for it and will let you know.

Reply:
Yeah, I just made 202 - like a mock. Thank you!